datafusion/physical-expr/src/aggregate/groups_accumulator/mod.rs

```diff
@@ -15,10 +15,18 @@
 // specific language governing permissions and limitations
 // under the License.
 
-pub(crate) mod accumulate;
 mod adapter;
-pub use accumulate::NullState;
+
```

Comment:
👍  for backwards compatibility

Maybe we can leave a comment explaining it is for backwards compatibility